src/movies.js

```diff
@@ -69,7 +69,37 @@ function orderByYear(a, b) {
 //console.log(movies);
 
 // Iteration 6: Alphabetic Order - Order by title and print the first 20 titles
-function orderAlphabetically() {}
+
+
+function orderAlphabetically(arr) {
+
+  const sortedMovies = movies.sort(function (a, b) {
+    if (a.title < b.title) {
+      return -1;
+    }
+    if (a.title > b.title) {
+      return 1;
+    }
+    return 0;
+  });
+
+console.log(sortedMovies)
+
+  const arrLength = sortedMovies.length;
+
+console.log(arrLength)
+
+  if (arrLength <= 3) {
+    return sortedMovies;
+  } else {
+    for (let i = 0; i > sortedMovies[3]; i++) {
+     return sortedMovies.pop();
+    }
+  }
+
+}
```

Comment:
1. Do not forget to have return at the end of the function
2. Function has to return array of sorted movie titles (not the array of objects)

So you might need to create new array with only movie titles, maybe use map method for that?